### PR TITLE
Mapping tooltips

### LINKED ISSUE #12341: some minor text and .pot errors
found some errors while translation

**texts did not found in .pot file**
collection filters - notice when add new rules
"You can't have more than 10 rules"
"You can't have more than 10 sort orders"

**texts did not found in .pot file, could be translated by adding them manually**
negadoctor: slider "D max"
lua scripts installer: "lua scripts installer"
lua scripts installer: "select action"
lua scripts installer: "select action to perform"
lua scripts installer: "install scripts"
lua scripts installer: "remind me later"
lua scripts installer: "don't show again"
lua scripts installer: "execute"
preferences - lua options: "lua scripts installer dont show again"
preferences - lua options: "do not show scripts_installer if lua scripts are not instelled"

**mouse hover descriptions always use the same text with tab names, could not be translated**
color zones tabs: lightness, saturation, hue
filmic rgb tabs: scene, reconstruct, look, display, options
negadoctor tabs: film properties, corrections, print properties
tone equalizer tabs: simple, advanced, masking
raw denoise tabs: all, R, G, B

**others**
color zones: LCh-C should be "chroma", but the module uses "saturation"
color balance rgb: use the same module description "affect color, brightness and contrast" with color balance, rough and could not separate two modules
color calibration: description of spot color mapping "use a color checker target to autoset CAT and channels" should belong to "calibrate with a color checker". and this description shows all over on target, lightness, hue and chroma in spot color mapping
filmic rgb: description of "use custom middle-gray values" in .pot claims c-format, but use "18.45 %%", if it's a c-format description, should use "%d%%"? or, it's a normal text should use "18.45%"

### PR DESCRIPTION
**Pull request by Nilvus**
Closes #12341 as this fix last thing showed by @HSUfineprint.

Tooltips strings are from actual manual. This PR also adds french translation. @TurboGit: I let update darktable.pot file.

### CODE REVIEW

File: src/iop/channelmixerrgb.c [hunk 1]
```diff
@@ -4075,7 +4075,7 @@ void gui_init(struct dt_iop_module_t *self)
      _("spot color mapping"),
      GTK_BOX(self->widget));
 
-  gtk_widget_set_tooltip_text(g->csspot.expander, _("use a color checker target to autoset CAT and channels"));
+  gtk_widget_set_tooltip_text(g->csspot.expander, _("define a target chromaticity (hue and chroma) for a particular region of the image (the control sample), which you then match against the same target chromaticity in other images. The control sample can either be a critical part of your subject or a non-moving and consistently-lit surface over your series of images."));
```
Comment (TurboGit) [inline src/iop/channelmixerrgb.c:4078]: The control -> the control

File: src/iop/exposure.c [hunk 2]
```diff
@@ -1080,6 +1080,8 @@ void gui_init(struct dt_iop_module_t *self)
      _("spot exposure mapping"),
      GTK_BOX(self->widget));
 
+  gtk_widget_set_tooltip_text(g->cs.expander, _("define a target brightness, in terms of exposure, for a selected region of the image (the control sample), which you then match against the same target brightness in other images. The control sample can either be a critical part of your subject or a non-moving and consistently-lit surface over your series of images."));
```
Comment (TurboGit) [inline src/iop/exposure.c:1083]: likewise

**Review by TurboGit — CHANGES_REQUESTED** [review]
fix casing

**Comment by Nilvus** [comment]
@TurboGit: casing fixed with last commit

**Pull request merged**